FileIO/VtkIO/VtuInterface.cpp

```diff
@@ -61,43 +70,34 @@ MeshLib::Mesh* VtuInterface::readVTUFile(std::string const &file_name)
 
 bool VtuInterface::writeToFile(std::string const &file_name)
 {
-	if(!_mesh)
+#ifdef USE_PETSC
+	// Also for other approach with DDC.
+	// In such case, a MPI_Comm argument is need to this member,
+	// and PETSC_COMM_WORLD shoud be replaced with the argument.  
+	int mpi_rank;
+	MPI_Comm_rank(PETSC_COMM_WORLD, &mpi_rank);
+	const std::string file_name_base = boost::erase_last_copy(file_name, ".vtu");
```

Comment:
What if the `file_name` variable does not end with vtu? Or is it guarantied somewhere else that the ending is `".vtu"`?